code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -28,6 +28,32 @@
 	var/obj/item/suppressor/S = new(src)
 	install_suppressor(S)
 
+/obj/item/gun/ballistic/automatic/pistol/der38
```

Comment:
As a difficult to code but potentially cool suggestion, being able to hide this inside a suit or something so you can sneak it into prison with you would be cool. Of course this is only a suggestion and doesn't need to be completed before merging as it may need us to make the suit attachment thing a component.

Reply:
I was debating on making a special set of plain black gloves that can store it, akin to a wrist sling or something. If I wanted to make it attachable to a jumpsuit, I could see about poking around with accessory code and if it's possible to make it not display on examine text.